app/views/jobs/_form.html.erb

```diff
@@ -1,15 +1,37 @@
-<%= form_for [@company, @job] do |f| %>
-  <%= f.label :title %>
-  <%= f.text_field :title %>
+<% if @job.errors.any? %>
+<h2><%= pluralize(@job.errors.count, "error") %> prohibited this record from being saved:</h2>
+<ul><% @job.errors.full_messages.each do |message| %>
+  <li><%= message %></li>
+  <% end %>
+  <% end %>
 
-  <%= f.label :description %>
-  <%= f.text_field :description %>
-
-  <%= f.label :level_of_interest %>
-  <%= f.number_field :level_of_interest %>
-
-  <%= f.label :city %>
-  <%= f.text_field :city %>
-
-  <%= f.submit %>
-<% end %>
+  <div>
+    <%= form_for [@company, @job] do |f| %>
+    <div class="form-group">
+      <%= f.label :title %>
+      <%= f.text_field :title, class:"form-control", placeholder: :title %>
+    </div>
+    <div class="form-group">
+      <%= f.label :description %>
+      <%= f.text_field :description, class:"form-control", placeholder: :description %>
+    </div>
+    <div class="form-group">
+      <%= f.label :level_of_interest %>
+      <%= f.number_field :level_of_interest,class:"form-control", placeholder: :level_of_interest %>
+    </div>
+    <div class="form-group">
+      <%= f.label :city %>
+      <%= f.collection_select :city_id, City.all, :id, :name %>
```

Comment:
we want to avoid querying our database in our views. ideally, you'd set `City.all` to an instance variable in your controller, then reference that here